.eslintrc.cjs

```diff
@@ -50,7 +50,14 @@ module.exports = {
   reportUnusedDisableDirectives: true,
 
   rules: {
-    '@typescript-eslint/prefer-ts-expect-error': 'error',
+    '@typescript-eslint/ban-ts-comment': [
+      'error',
+      {
+        // TODO tigthen to 'allow-with-description'
+        'ts-expect-error': false,
+        'ts-nocheck': false,
```

Comment:
Wondering if we should try to get rid of explicit `ts-nocheck` as much as possible. Maybe with some override config to disable checks wholesale where needed (e.g. cosmic-proto/codegen)

Reply:
Eventually, yeah. We have 114 files with it outside codegen. Thankfully 93 of those are tests. 14 of the non-test ones are in SwingSet. Thankfully 13 are in misc-tools.

Just 2 non-codegen `src` files, so we could it up for `src` pretty easily. Another time tho.